conventions/Documentation.md

```diff
@@ -1,20 +1,67 @@
 # Documentation
 
+## Stencil API reference
+
+Stencil uses [JSDoc](https://jsdoc.app) for their API reference generation. Stencil generates a [`docs-json`](https://stenciljs.com/docs/docs-json) output target, which is parsed and displayed on the [SDK site](https://developers.arcgis.com/calcite-design-system/components). The API reference includes property/attribute names, descriptions, types, values, and description notes (e.g. required, optional, deprecated). The SDK site updates the API reference after Calcite Component releases.
+
+### Style guide
+
+Follow these conventions when adding or editing API reference:
+
+- Use complete sentences with proper grammar, capitalization, and punctuation.
+- No abbreviations, e.g. use "property" instead of "prop".
+- Verbs must be in present tense.
+- Use the full tag name when referencing other Calcite Components (prefix with `calcite-`), e.g. `calcite-button` instead of `button`.
+- For plural context, use `calcite-button`s instead of `calcite-button` elements.
+- Use backticks (`` ` ``) for the names of slots, events, properties CSS variables, and component names (e.g. `calcite-button` instead of calcite-button).
```

Comment:
Additional comma?:

- Use backticks (`` ` ``) for the names of slots, events, properties[,] CSS variables, and component names (e.g. `calcite-button` instead of calcite-button).